Reminder-App/Reminder-App/Reminder.swift

```diff
@@ -8,13 +8,17 @@
 import Foundation
 
 struct Reminder {
+    
+    // MARK:- Instance Properties
     var title: String
     var dueDate: Date
     var notes: String? = nil
     var isComplete: Bool = false
 }
 
 extension Reminder {
+    
+    // MARK:- Static Properties
     static var testData: [Reminder] = [
         Reminder(title: "Create remote repo", dueDate: Date().addingTimeInterval(800.0), notes: "Use your own github account"),
```

Comment:
For now it is okay but Strings/text always keep in separate file named Globals/AppConstants
Make an enum in that file called Title/Strings and add these text as a variable
Like 
enum Titles {
static var createMessage = "Create remote repo"
}

Reply:
Okay. Will leave it for now because its just a dummy data.